composer.json

```diff
@@ -20,7 +20,10 @@
 		"ninjify/nunjuck": "^0.1.4",
 		"nette/tester": "^1.7.0",
 		"mockery/mockery": "^0.9.6",
-		"nette/di": "^2.4.4|^3.0"
+		"nette/di": "^2.4.4|^3.0",
+		"gamee/php-code-checker-rules": "^1.0",
```

Comment:
It's only for contributte/datagrid, we don't use it in other libs. Do you think it's needed?

Reply:
True, not needed. Updated in last commit